includes/wp-admin/class-admin.php

```diff
@@ -662,4 +664,28 @@ function ( $id ) {
 		</script>
 		<?php
 	}
+
+	/**
+	 * Add Dashboard widgets.
+	 */
+	public static function add_dashboard_widgets() {
+		\wp_add_dashboard_widget( 'activitypub_blog', __( 'ActivityPub Plugin News', 'activitypub' ), array( self::class, 'blog_dashboard_widget' ) );
+	}
+
+	/**
+	 * Add the ActivityPub.blog feed as a Dashboard widget.
+	 */
+	public static function blog_dashboard_widget() {
+		echo '<div class="rss-widget">';
+		\wp_widget_rss_output(
+			array(
+				'url'          => 'https://activitypub.blog/feed/',
+				'items'        => 10,
+				'show_summary' => 1,
```

Comment:
I think 10 would be fine if we didn't show summaries (which would be my slight preference). With summaries enabled, maybe we limit it to 3? I worry it might take up too much space otherwise.

Reply:
I would go with three posts with summary, now that we only have one post and we can switch to headlines later on.